building/build_options.adoc

```diff
@@ -0,0 +1,67 @@
+# Options for building JBoss Tools
+
+## Maven options
+
+Here are the options we use for JBoss Tools builds:
```

Comment:
Mostly valid complaints here on build_options.adoc,  but I'd just like to note that this file is incorrectly marked as new. It was simply a rename / reformat from .md to .adoc.  It should have been marked as 'moved' with no real changes to its content.